procedures/auxtel/observation_procedures/Parabolic_Focus_Sequence.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>Consider adding a line after this with the code to offset the telescope a bit. If the star falls in a bad place then we'll want to move it a little.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>